static/app/components/assigneeBadge.tsx

```diff
@@ -146,6 +143,12 @@ const TooltipWrapper = styled('div')`
   text-align: left;
 `;
 
+const StyledText = styled('div')`
+  color: ${p => p.theme.textColor};
+  max-width: 114px;
+  ${p => p.theme.overflowEllipsis};
+`;
```

Comment:
I’ve kept the `maxWidth` for Text applied in the `AssigneeBadge` to avoid having it grow too long; The 114px is for the text only, and it gets cut-off with an ellipsis now. The full width of the Tag is still 166px (icon on the left and chevron on the right)

Reply:
I remember I copied the 166px width from our old Tag to the ChonkTag, as I was afraid that it would break everything (obviously, there were no comments on why we picked 166px) 😓

Reply:
How about we only remove it in chonk so that we have some time to figure out behind the feature flag if it breaks anything?

Reply:
That, or we backport this change to the old tags, that way the behavior between the two tags is the same, and people who might still be using the old UI would see it behave the same way in both modes.

Reply:
> or we backport this change to the old tags

not fully sure what you mean? There is no old tag component anymore (just `core/badge/tag.tsx`) and it has 246 usages. I don’t wanna add this arbitrary `max-length` to all of them 😅 .

I went with only removing it in chonk: 365416d3